back/app/policies/nav_bar_item_policy.rb

```diff
@@ -1,4 +1,6 @@
 class NavBarItemPolicy < ApplicationPolicy
+  FEATURES_CODES = { 'initiatives' => 'proposals' }.freeze
```

Comment:
Double plural name, should be FEATURE_CODES instead? :-)

Reply:
I did it intentionally to show that the keys of the hash are features, and the values are codes.

Reply:
(I mean it's not 'feature codes', but 'features and codes')